ui-tests/src/test/java/org/openmrs/reference/RegisterPatientTest.java

```diff
@@ -33,9 +34,8 @@ public void tearDown() throws Exception {
         waitForPatientDeletion(patient.uuid);
     }
 
-    // Test for Story RA-71
     @Test
-    @Category(org.openmrs.reference.groups.BuildTests.class)
+    @Category(BuildTests.class)
```

Comment:
Refactored `@Category` annotation to align with the legacy ui test `convention`.